components/core/src/clp_s/InputConfig.cpp

```
@@ -467,4 +470,51 @@ void close_nested_readers(std::vector<std::shared_ptr<clp::ReaderInterface>> con
+                SPDLOG_WARN("Failed to create reader for {}, will retry.", path.path);
+                continue;
+            }
+            SPDLOG_ERROR("Failed to open input {} for reading.", path.path);
+            return {{}, FileType::Unknown};
+        }
+
+        auto result = try_deduce_reader_type(reader);
+        auto const& [nested_readers, file_type] = result;
+
+        if (FileType::Unknown == file_type
+            && NetworkUtils::is_retryable_curl_error(reader.get())
```

Cppcheck (2.20.0)

[style] The function 'wildcard_query_matches_any_encoded_var' is never used.

(unusedFunction)